app/containers/Pdf/PdfObjectFieldTemplate.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {View} from '@react-pdf/renderer';
+import {ObjectFieldTemplateProps} from 'react-jsonschema-form';
+
+const PdfObjectFieldTemplate: React.FunctionComponent<ObjectFieldTemplateProps> = props => {
+  return (
+    <>
+      {props.properties.map(
+        prop =>
+          prop.content && <View key={prop.content.key}>{prop.content}</View>
+      )}
+      {!props.properties && '\n'}
```

Comment:
The block above will fail if `props.properties` is undefined. So either this line is not needed, or the block above needs an extra check

Reply:
This was added (at some point) for formatting. I checked and it has no effect now... Removed.